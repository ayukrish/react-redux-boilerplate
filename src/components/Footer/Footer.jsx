import React from 'react';

const Footer = () => (
  <footer>
    footer here
  </footer>
);

export default Footer;
