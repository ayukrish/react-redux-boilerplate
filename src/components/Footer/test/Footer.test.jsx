import React from 'react';
// import { shallow } from 'enzyme';
// import { shallowToJson } from 'enzyme-to-json';
import renderer from 'react-test-renderer';

// Components
import Footer from '../Footer';


describe('Footer Test Suite', () => {
  it('renders correctly', () => {
    // const output = shallow(
    //   <Footer />,
    // );
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
    // expect(shallowToJson(output)).toMatchSnapshot();
  });
});
