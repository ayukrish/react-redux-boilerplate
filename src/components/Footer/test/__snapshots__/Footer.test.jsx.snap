// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Test Suite renders correctly 1`] = `
<footer>
  footer here
</footer>
`;
